qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -220,7 +222,7 @@ def __init__(
             tuple[str, complex],
             list[tuple[str, complex]],
             list[tuple[str, float]],
-            list[tuple[list[tuple[str, int]], complex]],
+            list[tuple[tuple[tuple[str, int], ...], complex]],
```

Comment:
This is API breaking change of the type hint. I have learned Python's type very recently.
If you use `collections.abc.Sequence` or `collections.abc.Iterable`, you can not break because list and tuple are subclass of them.
Sequence probably does not require any code changes, but Iterable is broader, so Iterable is better if it's consumed somewhere.

Reply:
Done.